docs/dev/python/index.md

```diff
@@ -22,3 +23,43 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
+<img src="{% link /fig/pyflink.svg %}" alt="PyFlink" class="offset" width="50%" />
+
+PyFlink is a language for building unified batch and streaming workloads.
+This means real-time streaming pipelines, performing exploratory data
+analysis at scale, building machine learning pipelines, and creating ETLs for a data platform.
+If you're already familiar with Python and libraries such as Pandas, then PyFlink makes it simple
+to leverage the full capabilities of the Apache Flink ecosystem.
+
+The PyFlink Table API makes it simple to write powerful relational queries for building reports and
+ETL pipelines.
+At the same time, the PyFlink DataStream API gives developers access to low-level control over
+state and time, unlocking the full power of stream processing.
```

Comment:
```suggestion
* The **PyFlink Table API** allows you to write powerful relational queries in a way that is similar to using SQL or working with tabular data in Python.

* At the same time, the **PyFlink DataStream API** gives you lower-level control over the core building blocks of Flink ([state](https://ci.apache.org/projects/flink/flink-docs-release-1.11/concepts/stateful-stream-processing.html#what-is-state) and [time](https://ci.apache.org/projects/flink/flink-docs-release-1.11/concepts/timely-stream-processing.html#introduction)) to build more complex stream processing use cases.
```

Reply:
Think it's important here to give users something to identify with, like SQL and Pandas. Also, I don't think state and time in the way that we think about them are straightforward for Python users that don't know Flink.

Reply:
I like that. For future reference, we don't cross-link via the actual url state should be: 

```md
[state]({% link concepts/stateful-stream-processing.md}#what-is-state)
```